src/app.jsx

```diff
@@ -305,10 +305,7 @@ class AppActive extends React.Component {
                 }
                 {path.length > 0 && path[0] == 'networks' &&
                 <NetworkList networks={networks}
-                    resourceHasError={this.state.resourceHasError}
-                    vms={vms}
-                    nodeDevices={nodeDevices}
-                    interfaces={interfaces} />
+                             resourceHasError={this.state.resourceHasError} />
```

Comment:
Just a question, but why hasn't similar cleanup been done for other components? e.g. storagePools, networks, nodeDevice are also passed down to <VmDetailsPage>, <HostVmsList>, and vms array is passed to <StoragePoolList>, etc... 
I'm however fine with doing that in a followup, just asking if there is any problem which prevents similar cleanups for other components.

Reply:
> Just a question, but why hasn't similar cleanup been done for other components?

Because it would have been more work, and wasn't necessary to keep the dialog behavior unchanged.  If you all are happy with this direction, I can work more on this.

Reply:
I'm fine with doing that in the future, as that can be split to a different PR